nemo/collections/asr/models/aed_multitask_models.py

```diff
@@ -603,50 +578,33 @@ def forward(
 
         return transf_log_probs, encoded_len, enc_states, enc_mask
 
-    def compute_loss(
-        self, batch: tuple[torch.Tensor, torch.Tensor, torch.Tensor, torch.Tensor] | None
-    ) -> torch.Tensor:
-        """
-        Run forward pass through the model and compute the loss.
-
-        Args:
-            batch: a tuple of 4 tensors (signal, signal_len, tokens, tokens_len) as returned
-                by :class:`~nemo.collections.asr.data.audio_to_text_lhotse_prompted.PromptedAudioToTextLhotseDataset`.
-                When batch is ``None``, we'll return a zero tensor.
-        Returns:
-            The computed loss value as a single-element tensor.
-        """
+    # PTL-specific methods
+    def training_step(self, batch, batch_nb):
+        signal, signal_len, transcript, transcript_len = batch
 
         if batch is None:
-            return torch.tensor([0.0])
+            tensorboard_logs = {
+                'train_loss': torch.Tensor[0.0],
+                'learning_rate': self._optimizer.param_groups[0]['lr'],
+            }
+            return {'loss': audio_loss, 'log': tensorboard_logs}
```

Comment:
using audio_loss before defining?

Reply:
The merge conflict messed up this line. Should be fixed now.